WTWSMS/common/landed_titles/WtWSMS_kingdoms.txt

```diff
@@ -1111,7 +1113,22 @@ d_juhuria = {
 	
 	allow = {
 		always = no
+#		OR = {
+#			AND = {
+#				culture = juhuro
+#				religion = tzadikite
+#				completely_controls = c_kakheti
+#			}
+#			AND = {
+#				religion = tzakikite
+#				completely_controls = c_kakheti
+#				higher_real_tier_than = KING
+#				realm_size = 75
+#			}
+#		}		
```

Comment:
Adding commented out notes for yourself to the main mod doesn't serve much of a purpose at this point. I'd suggest that you keep this in store on your end if you still need it.